vmaas/types.go

```diff
@@ -167,23 +178,26 @@ type RepoDetail struct {
 	ThirdParty bool
 }
 
+type CveDetailCommon struct {
+	Cvss2Score    *string  `json:"cvss3_score"`
+	Cvss2Metrics  *string  `json:"cvss3_metrics"`
+	Cvss3Score    *string  `json:"cvss2_score"`
+	Cvss3Metrics  *string  `json:"cvss2_metrics"`
+	CWEs          []string `json:"cwe_list"`
+	Description   string   `json:"description"`
+	Impact        string   `json:"impact"`
+	ModifiedDate  *string  `json:"modified_date"`
+	PublishedDate *string  `json:"public_date"`
+	RedHatURL     *string  `json:"redhat_url"`
+	SecondaryURL  *string  `json:"secondary_url"`
```

Comment:
I checked the response of current vmaas implementation and we need to use `string` instead of `*string` to have the same response, there is an empty string for a missing value.
API spec https://github.com/RedHatInsights/vmaas/blob/master/vmaas/webapp/webapp.v3.spec.yaml have these values also as `string` and not nullable.
We can probably use `time.Time` for `ModifiedDate` and `PublishedDate` so we don't need to convert time.Time into string and json.Marshal in RedHatInsights/vmaas will do that for us.

Most likely you'd need to update also `loadCves` func in load.go file. It might fail if the value is stored as NULL in sqlite. It should be enough to change `rows := getAllRows("cve_detail", "*")` and instead of `*` list all fields. For fields that can be NULL, use COALESCE(field_name, ''). Make sure that fields are in the same order as they are read by `rows.Scan`. Currently cvss3_score and cvss2_score are switched up.